.github/workflows/build_wheels.yml

```diff
@@ -81,34 +80,33 @@ jobs:
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.16.2
         env:
-          CIBW_BUILD: ${{ matrix.cibw_python }}
-          CIBW_ARCHS_LINUX: ${{ matrix.cibw_arch }}
+          CIBW_BUILD: "cp3{7..12}-${{ matrix.wheel_type }}"
+          CIBW_ARCHS_LINUX: auto aarch64
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_TEST_EXTRAS: test
           CIBW_TEST_COMMAND: python -m pytest {package}/tests
           CIBW_TEST_SKIP: "*aarch64*"
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
-          name: dist
+          name: ${{ matrix.wheel_type }}-wheels
           path: ./wheelhouse/*.whl
 
   build_wheels_macos:
     needs: [build_sdist]
-    name: Wheel for MacOS-${{ matrix.cibw_python }}-${{ matrix.cibw_arch }}
+    name: ${{ matrix.cibw_arch }} wheels
```

Comment:
```suggestion
    name: macosx_${{ matrix.cibw_arch }} wheels
```

Reply:
Without these changes, we see the following in PR checks:

```
Wheels / x86_64 wheels (pull_request)
Wheels / arm64 wheels (pull_request)
Wheels / manylinux_x86_64 wheels (pull_request)
Wheels / manylinux_aarch64 wheels (pull_request)
```

I think this is confusing.